xgboost_ray/main.py

```diff
@@ -7,21 +7,30 @@
 
 from threading import Thread
 
-from ray.exceptions import RayActorError
-
 try:
     import ray
     from ray import logger
     from ray.services import get_node_ip_address
+    from ray.exceptions import RayActorError
     from ray.util.queue import Queue
+    from ray.actor import ActorHandle
     RAY_INSTALLED = True
 except ImportError:
     ray = None
     logger = None
     get_node_ip_address = None
     Queue = None
+    ActorHandle = None
     RAY_INSTALLED = False
 
+# Tune imports.
+try:
+    from ray import tune
+    from xgboost_ray.tune import RayTuneReportCallback
+    TUNE_INSTALLED = True
+except ImportError:
+    TUNE_INSTALLED = False
```

Comment:
```suggestion
    tune = None
    RayTuneReportCallback = None
    TUNE_INSTALLED = False
```
so that IDEs won't complain